crates/viewer/re_dataframe_ui/src/display_record_batch.rs

```diff
@@ -232,10 +251,12 @@ impl DisplayComponentColumn {
                     row_id = Some(RowId::from_u128(
                         quick_partial_hash(buffer, SECTION_LENGTH).hash64() as _,
                     ));
+
+                    variant_name = Some(VariantName::from(REDAP_THUMBNAIL_VARIANT));
```

Comment:
Maybe this entire block should be conditioned on the new new `is_image()`. Also, setting the `variant_name` should not be conditioned on the `row_id` being initially `None`.